esp/esdllib/README.md

```diff
@@ -781,7 +1110,7 @@ then execute this entry point script:
 
 ```xml
 <es:BackendRequest xmlns:es="urn:hpcc:esdl:script">
-    <es:set-value target="MyFirstValue" value="getDataSection(&apos;mySection&apos;)/firstValue"/>
+    <es:set-value target="MyFirstValue" value="getDataSection('mySection')/firstValue"/>
```

Comment:
I found myself going to the code to see the difference between `@select` and `@value`. There is none beyond order of precedence.

- Should they be interchangeable?
- Should `@value` be documented as supported for legacy purposes?
- Should `@value` be treated as a literal and not an path?
- Should `@value` not be documented?

Reply:
I've changed to use `@select`. I think your other good questions should be resolved when we unify the style of the documentation.